packages/lesswrong/server/repos/VotesRepo.ts

```diff
@@ -496,7 +496,27 @@ class VotesRepo extends AbstractRepo<"Votes"> {
     );
     return results.map(({ vote_id }) => vote_id);
   }
-
+  
+  async getVotesOnSamePost({userId, postId, excludedDocumentId}: {
+    userId: string,
+    postId: string,
+    excludedDocumentId: string,
+  }): Promise<DbVote[]> {
+    return await this.manyOrNone(`
+      SELECT
+        v.*
+      FROM
+        "Votes" v,
+        "Comments" c
```

Comment:
Is this _implicitly_ a join with the `AND c._id = v."documentId"`?  For consistency & readability I bid that it be explicitly a join